lib/vector-core/src/event/vrl_target.rs

```diff
@@ -65,7 +65,7 @@ impl VrlTarget {
 }
 
 impl vrl_core::Target for VrlTarget {
-    fn insert(&mut self, path: &LookupBuf, value: vrl_core::Value) -> Result<(), String> {
+    fn target_insert(&mut self, path: &LookupBuf, value: vrl_core::Value) -> Result<(), String> {
```

Comment:
These renames are unfortunate. What existing implementations did they collide with? (trying to think of what other `get`/`insert`/`remove` functions there would be on `Value`)

Reply:
These came from the Vector `Value` type, and are directly implemented. There were tons of conflicts for these.

get: https://github.com/vectordotdev/vector/blob/502a3e7591fdbd2ed65be8419be873aa4568b94e/lib/value/src/value.rs#L694-L697

insert: https://github.com/vectordotdev/vector/blob/502a3e7591fdbd2ed65be8419be873aa4568b94e/lib/value/src/value.rs#L451-L455

remove: https://github.com/vectordotdev/vector/blob/502a3e7591fdbd2ed65be8419be873aa4568b94e/lib/value/src/value.rs#L545-L549

Reply:
We might be able to merge these in the future, but that's out of scope for now.

Reply:
Ah yeah, I forgot about that API in Vector.